apps/elasticsearch/deployers/run_litmus_test.yml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-elasticsearch-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: elasticsearch-deployment-litmus
+
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays
+            value: default
+
+          - name: PROVIDER_STORAGE_CLASS
+            # Supported values: openebs-standard,openebs-standalone,local-storage
+            value: openebs-standard
+
+            # Application pvc
+          - name: APP_PVC
+            value: openebs-esearch 
+
+            # Application label
+          - name: APP_LABEL
+            value: 'app=elasticsearch'
+
+            # Application namespace
+          - name: APP_NAMESPACE
+            value: app-esearch-ns 
+
+          - name: DEPLOY_TYPE
+            value: statefulset
+
+          - name: APP_REPLICA
+            value: 'replicas=1'
+            
+          # Use 'deprovision' for app-clean up
+          - name: ACTION
+            value: provision
```

Comment:
Can you test the deletion also and paste the output.

Reply:
@gprasath yes i have tested this for deprovisioning also in both  jiva and cstor data plains.i will paste the output.